int16 val
int16 refreshRate
bool  streamEnabled
bool enabled
